modules/jaeger-sidecar-manual.adoc

```diff
@@ -26,12 +26,12 @@ spec:
     spec:
       containers:
         - name: example-app
-          image: jaegertracing/vertx-create-span:operator-e2e-tests
```

Comment:
This image can be included, as is a test app - although possibly refer to the same `acme/myapp:myversion` value as above.